ChangeLog

```diff
@@ -1,401 +1,29 @@
-# 2020-07-20 Version 2.2.0
-
-Important notes:
-* CVE-2020-15103 - Integer overflow due to missing input sanitation in rdpegfx channel
-
-Noteworty changes:
-* fix: memory leak in nsc
-* urbdrc
-  * some fixes and improvements
-* build
-  *  use cmake to detect getlogin_r
-  * improve asan checks/detection
-* server/proxy
-  * new: support for heartbeats
-  * new: support for rail handshake ex flags
-  * fix: possible race condition with redirects
-
-Fixed issues:
-* #6263 Sound & mic - filter GSM codec for microphone redirection
-* #6335: windows client title length
-* #6370 - "Alternate Secondary Drawing Order UNKNOWN"
-* #6298 - remoteapp with dialog is disconnecting when it loses focus
-* #6299 - v2.1.2: Can't connect to Windows7
+# 2023-07-17 Version 3.0.0-beta1
+
+We are pleased to announce the first beta release for the next stable 3.0
+series of FreeRDP. It has been a huge endeavour to implement all the new
+shiny bells and whistles as well as clean up the code base and we´re still
+ironing out some smaller glitches.
+This is the first API breaking change since the 2.0 series and there are
+some adjustments to be made for existing applictaions.
+See https://github.com/FreeRDP/FreeRDP/wiki/FreeRDP3-migration-notes for
+help (still incomplete)
+
+Noteworthy changes:
+* Support for PAA authentication
+* Support for websocket transport
+* Support smartcard authentication (TLS and NLA)
+* Full smartcard emulation support (login with certificate + key)
+* Rewritten proxy, new module API
+* New reference client based on SDL2 (work in progress)
+* Full OpenSSL 3 support
+* Updated RDP protocol support
+* Improved xfreerdp remote app support
```

Comment:
Didn't you add EnhancedRemoteApp (using GFX here (MapSurfaceToWindow), etc.)? If yes, doesn't avoid this painting black rectangles on the screen or allow now having shadows of RAIL windows drawn?